src/ApiClient.php

```diff
@@ -3,6 +3,7 @@
 namespace OneFit\GraphQLClient;
 
 use GuzzleHttp\ClientInterface;
+use Illuminate\Support\Arr;
```

Comment:
If you move this to the Service provider then this class should be easier to test and have less static dependencies.

Reply:
Arr class is part of Illuminate package of laravel it's an helper class, and suppose to be use this way. As the helper class injected using the composer https://github.com/laravel/framework/blob/5.8/composer.json#L97
So we don't need to inject it separately

Reply:
My bad @ricardofiorani the above explanation was the wrong explanation to you comment. I Agree with you, static dependency we should avoid, only if those you wanted to mock. However i don't want to mock Arr at all, its part of Illuminate/Support ,where it already have its test cases handled. And I wanted to use as it is, even for my test.